locales/hi.ftl

```diff
@@ -0,0 +1,20 @@
+title = Firefox Quantum Release कैंपेन
+project-name = Firefox Quantum Release
+main-title = Firefox का पलटवार
+main-description = नये Firefox के बारे में लोगो को बतायें। 14 नवम्बर को शाम साढ़े-सात बजे आइए ऑनलाइन और ट्वीट करिए अपनी मनपसंद राय नए Firefox के बारे में। साथ में #newfirefox लगाना बिलकुल मत भूलियेगा। आप Firefox के ट्वीट्स को रीट्वीट भी कर सकते हैं।
+thunderclap-cta = मदद करें Firefox Quantum की। 
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = आधिकारिक ट्वीट को रीट्वीट करें
+tweet-1 = नए #Firefox को उपयोग में लाएं और जाने की कैसे ये पहले से दो गुना ज्यादा तेज़ बन गया है। 💥 अभी Download करे : https://mzl.la/newfirefox
```

Comment:
I'm still missing the #newfirefox hashtag here. This is crucial (see the description ;)). Here maybe instead of the #Firefox hashtag?